environment.yml

```diff
@@ -21,13 +21,13 @@ dependencies:
   - psycopg2
   - sqlalchemy<2.0  # v2.0 does not supported by some packages yet.
   - geoalchemy2
-  - scrapy
+  - scrapy==2.11.1
   - dask
   - bokeh!=3.0.*,>=2.4.2  # for Dask dashboard
   - laspy
   - pip:
     - laszip  # for vertical datum transference
-    - geofabrics
+    - geofabrics<=0.10.23
     - geovoronoi
```

Comment:
I feel like `geovoronoi` isn't used

Reply:
I am not sure. A question for Xander @xandercai